packages/block-library/src/accordion-heading/edit.js

```diff
@@ -28,14 +36,27 @@ export default function Edit( { attributes, setAttributes, context } ) {
 		}
 	}, [ iconPosition, showIcon, setAttributes ] );
 
+	useEffect( () => {
+		if ( isSelected ) {
+			const timeoutId = setTimeout( () => {
+				titleRef.current?.focus();
+			}, 0 );
+
+			return () => {
+				clearTimeout( timeoutId );
+			};
+		}
+	}, [ isSelected ] );
+
 	const blockProps = useBlockProps();
 	const spacingProps = useSpacingProps( attributes );
 
 	return (
 		<TagName { ...blockProps }>
 			<button
 				className="wp-block-accordion-heading__toggle"
-				style={ spacingProps.style }
+				style={ { ...spacingProps.style } }
```

Comment:
```suggestion
				style={ spacingProps.style }
```

Nit: No need to change this line